src/starcheck.pl

```diff
@@ -656,6 +679,27 @@ sub json_obsids{
     $out .= qq{${font_stop}\n};
 }
 
+
+# Run independent attitude checker
+my $CHAR_REQUIRED_AFTER = '2015:315:00:00:00.000';
+if ((defined $char_file) or ($bs[0]->{time} > date2time($CHAR_REQUIRED_AFTER))){
+    $out .= "------------  VERIFY TARGETS (SI_ALIGN CHECK)  -----------------\n\n";
+    if (not defined $char_file){
+        $out .= "Error.  Characteristics file not found. \n";
+    }
+    else{
+        my $att_report = "${STARCHECK}/pcad_att_check.txt";
+        my $att_ok = make_pcad_attitude_check_report(
```

Comment:
BTW, our previous discussion about parse_cm and continuity *should* be beside the point.  What matters is that it seems this call is failing from within Perl, but succeeding when called exactly the same way from IPython.  That's the place to start debugging.